docs/dev_guide/migration/debug-with-codepin.rst

```diff
@@ -349,12 +358,30 @@ Following is an example of the analysis report.
 .. code-block::
 
     CodePin Summary
-    Totally APIs count, 2
-    Consistently APIs count, 2
-    Most Time-consuming Kernel(CUDA), example.cu:26:3:epilog, time:8.2316
-    Most Time-consuming Kernel(SYCL), example.cu:26:3:epilog, time:10.2575
-    Peak Device Memory Used(CUDA), 508100608
-    Peak Device Memory Used(SYCL), 31023112192
+    Total API count, 2
+    Consistent API count, 0
+    Most Time-consuming Kernel(CUDA), vectorAdd:example.cu:24:5:epilog, time:16.8069
+    Most Time-consuming Kernel(SYCL), vectorAdd:example.cu:24:5:prolog, time:18.3240
+    Peak Device Memory Used(CUDA), 445644800
+    Peak Device Memory Used(SYCL), 540689534976
     CUDA Meta Data ID, SYCL Meta Data ID, Type, Detail
-    example.cu:26:3:prolog,example.cu:26:3:prolog,Data value,[WARNING: METADATA MISMATCH] The pair of prolog data example.cu:26:3:prolog are mismatched,
-    and the corresponding pair of epilog data matches. This mismatch may be caused by the initialized memory or argument used in the API example.cu.
\ No newline at end of file
+    vectorAdd:example.cu:24:5:epilog,vectorAdd:example.cu:24:5:epilog,Data value,The location of failed ID Errors occurred during comparison: d_a->"Data"->[3]->"Data"->[0]->"x"->"Data"->[0] and [ERROR: DATA VALUE MISMATCH] the CUDA value 1 differs from the SYCL value 26518016.; d_result->"Data"->[3]->"Data"->[0]->"x"->"Data"->[0] and [ERROR: DATA VALUE MISMATCH] the CUDA value 2 differs from the SYCL value 26518017.
+    vectorAdd:example.cu:24:5:prolog,vectorAdd:example.cu:24:5:prolog,Data value,[WARNING: METADATA MISMATCH] The pair of prolog data vectorAdd:example.cu:24:5:prolog are mismatched, and the corresponding pair of epilog data matches. This mismatch may be caused by the initialized memory or argument used in the API vectorAdd.
+
+Data Flow Graph  
+~~~~~~~~~~~~~~~
+
+codepin-report.py can generate a data   flow graph for 
+kernels with option ``--generate-data-flow-graph``.  The data flow graph presents visualizations of kernel execution and compares results between CUDA and SYCL, highlighting the execution mismatch between CUDA and SYCL code.
+In the data flow graph, each kernel execution and its input and output arguments are grouped into a layer, presenting a run status of the kernel execution. The value   of input and output arguments are tagged with version information in the form of “V<num>”. For example,   the initial version is tagged as V0, and once the value of the argument is updated, the version number will be increased. For a specific kernel execution, if there’s a mismatch between CUDA and SYCL results, the mismatched argument node will be colored red.
+
+.. figure:: DataFlowGraph.png
```

Comment:
The path should be ../../_images/DataFlowGraph.png

Reply:
Updates made, please review & approve & merge if it looks ok. Thanks!